_pages/install/browser-support.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Web Browser Support
+---
+
+Spartacus libraries are designed to take advantage of the latest web platform standards, while also allowing you to run your Spartacus storefront on as many different web browsers as possible. However, some older browsers do not support the latest standards, and as a result, these browsers are not supported by Spartacus. Officially, Spartacus only supports evergreen browsers.
+
```

Comment:
Can we add here the reasoning why we only support evergreen browsers? 

- we follow standard Angular
- we do not want to invest too much in (automated) QA in very old browsers
- we like to keep Spartacus as clean as possible and avoid _browser quirks_ in the library code. 

In addition, we can sum up the techniques we expect application  using: 
- typescript; transpile to older JS versions
- postcss; automatically add CSS vendor prefixes where needed in an after process
- polyfills; add JS files that fill in any missing browser features

Reply:
Pretty much what you had already, however these ^^ techniques are not only for IE11. Its common sense to use those for any browser that might have a (small or large) gap.

Reply:
Done.